include/enrico/openmc_nek_driver.h

```diff
@@ -47,9 +48,9 @@ class OpenmcNekDriver : public CoupledDriver {
   bool is_converged() override;
 
   Comm intranode_comm_; //!< The communicator representing intranode ranks
-  std::unique_ptr<OpenmcDriver> openmc_driver_; //!< The OpenMC driver
-  std::unique_ptr<NekDriver> nek_driver_;       //!< The Nek5000 driver
-  int openmc_procs_per_node_; //!< Number of MPI ranks per (shared-memory) node in OpenMC comm
+  double pressure_;                             //!< System pressure in [MPa]
```

Comment:
Looks like the `pressure_` member crept back in during your rebase. Can you remove this?